packages/flutter/lib/src/widgets/text.dart

```diff
@@ -284,7 +313,7 @@ class Text extends StatelessWidget {
       overflow: overflow ?? defaultTextStyle.overflow,
       textScaleFactor: textScaleFactor ?? MediaQuery.of(context, nullOk: true)?.textScaleFactor ?? 1.0,
       maxLines: maxLines ?? defaultTextStyle.maxLines,
-      text: new TextSpan(
+      text: span ?? new TextSpan(
```

Comment:
if you do this, DefaultTextStyle isn't going to apply, which is going to defeat the entire purpose of this patch. :-)

Reply:
Ohhhhh, okay, I think figured out how this is supposed to work